login/template.ftl

```diff
@@ -29,6 +29,19 @@
             <script src="${script}" type="text/javascript"></script>
         </#list>
     </#if>
+    <script type="text/javascript">
+        var _paq = window._paq || [];
+        _paq.push(['trackPageView']);
+        _paq.push(['enableLinkTracking']);
+        _paq.push(["setUserId", "REPLACE THIS WITH THE USER ID, IF KNOWN"]);
+        (function() {
+            var u="https://piwik.cirg.washington.edu/";
+            _paq.push(['setTrackerUrl', u+'matomo.php']);
+            _paq.push(['setSiteId', '35']);
```

Comment:
@mcjustin would we ever want the site ID configurable for different deploys of cosri?

Reply:
Yes - @achen2401  I should have clarified, the matomo ID (35 here) should be configurable per deployment.

Reply:
@mcjustin & @ivan-c I made the change so matomo ID is obtained via configuration variable.  Will you review?  As I don't have my own instance set up for keycloak.  Will one of you try this when you have a minute and see if it actually works?

Reply:
I'll give it a shot. Noting that I'll change this in matomo to label site ID 35 as COSRI test/dev - not prod.

Reply:
@ivan-c  how do I try out this branch at https://keycloak.mcjustin.dcw.dev.cirg.uw.edu ? Pecking around at `flerovium:/home/mcjustin/dcw-environments/dev` I didn't find anything obvious...